### DISCUSSION TITLE
Using only the brick on M1 once desired charge has been reached?

### DISCUSSION THREAD

**Discussion by laksheybco** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#274] [General]
Hello! Thanks for making something excellent and as specific as this for those who care about their battery health. 

Just wanted to know, does this currently or is there a way such that once the desired state of charge has been reached, the computer will only rely on the brick and not draw any power to the battery. In other words, the cycle count won't be changed since the battery is essentially being left alone?

Thanks again

**Comment by MatthiasKerbl** [comment]
> Hello! Thanks for making something excellent and as specific as this for those who care about their battery health.
> 
> Just wanted to know, does this currently or is there a way such that once the desired state of charge has been reached, the computer will only rely on the brick and not draw any power to the battery. In other words, the cycle count won't be changed since the battery is essentially being left alone?
> 
> Thanks again

Hi @laksheybco,

once your MacBook has reached the charge limit, it will stop using the battery and only uses the power brick.
